tools/dependency/validate.py

```diff
@@ -221,8 +219,9 @@ def ValidateExtensionDeps(self, name, target):
       metadata = self._dep_info.GetMetadata(d)
       if metadata:
         use_category = metadata['use_category']
-        if ('dataplane_ext' not in use_category and 'observability_ext' not in use_category and
-            'other' not in use_category):
+        valid_use_category = ('dataplane_ext' in use_category or
```

Comment:
nit: conjunction at the beginning of lines

i think even pep8 is reversing this